src/v2/Apps/PriceDatabase/Components/ArtistAutosuggest.tsx

```diff
@@ -0,0 +1,165 @@
+import Autosuggest from "react-autosuggest"
+import React, { useEffect, useState } from "react"
+import {
+  Input,
+  MagnifyingGlassIcon,
+  usePosition,
+  Text,
+  Clickable,
+} from "@artsy/palette"
+import { graphql } from "lib/graphql"
+import { fetchQuery } from "relay-runtime"
+import { ArtistAutosuggest_SearchConnection_Query } from "v2/__generated__/ArtistAutosuggest_SearchConnection_Query.graphql"
+import { useSystemContext } from "v2/System"
+import { Container } from "v2/Components/Sticky"
+
+interface ArtistAutosuggestProps {
+  onChange?: (artustSlug: string) => void
+}
+
+export const ArtistAutosuggest: React.FC<ArtistAutosuggestProps> = ({
```

Comment:
Should we move this component to Palette or just leave it here? 🤔

Reply:
An autocomplete would be a good thing — I've been meaning to build one to replace the primary nav bar search. It'd have to be accessible; do keyboard nav, etc. I'll see if Jonathan has any designs that could encompass all of the use cases.

Reply:
Very much wanting to get rid of `react-autocomplete` as well; which is partially the motivation.

Reply:
Sounds good! Then I'll leave it here until we can replace it.